recipe/meta.yaml

```diff
@@ -1,13 +1,13 @@
 {% set name = "tornado-sqlalchemy" %}
-{% set version = "0.7.0" %}
+{% set version = "0.8.0" %}
 
 package:
   name: "{{ name|lower }}"
   version: "{{ version }}"
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: f37e59c376aa7661beb96992977c64e6b0baf5cbe74c41ccc0bf3b25dee0e637
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name.replace('-', '_') }}-{{ version }}.tar.gz
```

Comment:
This line changes the name of the package

Reply:
It changes the name of the package as downloaded from PyPI, but not the name of the resulting package... the package on PyPI is now really named `tornado_sqlalchemy-0.8.0.tar.gz` so this is correct it seems:

![image](https://github.com/conda-forge/tornado-sqlalchemy-feedstock/assets/1085180/6dbdaf5e-9b87-409e-8445-6f7a94c1e31f)